docs/source/inference.mdx

```diff
@@ -96,13 +96,13 @@ tokenizer.save_pretrained(save_directory)
 
 ### Weight only quantization
 
-You can also apply INT8 quantization on your models weights when exporting your model with the CLI:
+You can also apply 8-bit or 4-bit weight quantization when exporting your model with the CLI:
```

Comment:
Could we mention that this is primarily useful for specific model types (CausalLM, ...) with a large number of weights?

Reply:
Addressed